gke/provision-gke.sh

```diff
@@ -27,7 +27,7 @@ if [ "${EXISTING_CLUSTER}" == "${CLUSTER_NAME}" ]; then
     echo "Cluster already created."
 else
     echo "Creating cluster..."
-    gcloud beta container clusters create ${CLUSTER_NAME} --zone ${CLUSTER_ZONE} \
+    gcloud beta container clusters create $CLUSTER_NAME --zone $CLUSTER_ZONE \
```

Comment:
I used curly braces in variable name here to keep it consistent. Was this an intentional change as it is not needed?

Reply:
Not intentional, will fix it.